sodiumpp/test.cpp

```diff
@@ -135,7 +135,7 @@ go_bandit([](){
               ls3 = std::move(ls2); // move assigment
               const char* const ls3_data_ptr = &ls3[0];
               AssertThat(ls2_data_ptr == ls3_data_ptr, Equals(true));
-        });
+        });*/
```

Comment:
this part of our tests do not make sense now until API is clarified re moving/SSO/constructing only from locked string